lib/maze/client/appium/base_client.rb

```diff
@@ -86,6 +91,15 @@ def start_driver(config)
           end
         end
 
+        def start_scenario
+          # Write Maze's address to file and push to the device
+          Maze::Api::Appium::FileManager.new.write_app_file("{\"maze_address\": \"#{maze_address}\"}",
+                                                            FIXTURE_CONFIG)
```

Comment:
Might be nicer to avoid escaping the quotes, e.g.:

```suggestion
          Maze::Api::Appium::FileManager.new.write_app_file(
            JSON.generate({ maze_address: maze_address }),
            FIXTURE_CONFIG
          )
```

or:

```suggestion
          Maze::Api::Appium::FileManager.new.write_app_file(
            %Q({ "maze_address": #{maze_address} }),
            FIXTURE_CONFIG
          )
```

Reply:
Agreed.  I wasn't keen on it when I did it, but didn't stop long enough to consider an alternative.